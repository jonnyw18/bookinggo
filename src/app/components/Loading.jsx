import React from 'react';
import ReactLoading from 'react-loading';

const Loading = ({ type, color }) => (
    <ReactLoading type={'balls'} color={color} height={667} width={375} />
);

export default Loading;