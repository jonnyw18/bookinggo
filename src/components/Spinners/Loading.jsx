import React from 'react';

const Loading = () =>
    <img src="https://cdn2.rcstatic.com/images/site_graphics/newsite/preloader64.gif" className="loading-spinner" alt="loading spinner"/>

export default Loading;