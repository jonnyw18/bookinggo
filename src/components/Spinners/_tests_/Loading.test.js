import Loading from '../Loading';

describe('<Loading />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
