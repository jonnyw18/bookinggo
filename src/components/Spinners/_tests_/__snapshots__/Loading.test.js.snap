// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders correctly 1`] = `
<img
  alt="loading spinner"
  className="loading-spinner"
  src="https://cdn2.rcstatic.com/images/site_graphics/newsite/preloader64.gif"
/>
`;
